### DISCUSSION TITLE
Hard coded colors

### DISCUSSION THREAD

**Discussion by BeBel42** [bladewindui/ui#197] [Ideas]
I saw that you removed the hard coded colors from the buttons. I really think it is a *good* idea!
Are you going to do the same for the other components? Such as the checkboxes for instance?
It's true that it would be much nicer for the user to not think about the "Nine availble colors", especially if "primary" is supported (because primary can already be anything, so why restrict the others?)

**Comment by mkocansey** [comment]
Yep! With time I'll do the same for the other components as well. To get rid of the hard coded colours. I think the earlier thinking was more to show users how the components will look in other colours, then colour options stuck.
